src/MCRun.jl

```diff
@@ -290,48 +593,95 @@ function ptmc_run!(mc_states, move_strat, mc_params, pot, ensemble, results)
     println("Total number of moves per MC cycle: ", n_steps)
     println()
 
+    
     #equilibration cycle
-    for i = 1:mc_params.eq_cycles
-        @inbounds mc_states = mc_cycle!(mc_states, move_strat, mc_params, pot, ensemble, n_steps, a, v, r)
-        if rem(i, mc_params.n_adjust) == 0
-            for i_traj = 1:mc_params.n_traj
-                update_max_stepsize!(mc_states[i_traj], mc_params.n_adjust, a, v, r)
-            end 
+    if restart == false
+        #this initialises the max and min energies for histograms
+        if save_ham == false
```

Comment:
absurdly high and low values are used for initialising the min/max energies. This is required for initialising the histograms